packages/evm/contracts/court/controller/Controller.sol

```diff
@@ -45,9 +46,13 @@ contract Controller is IsContract, Modules, CourtClock, CourtConfig {
     // List of all modules registered for the system indexed by address
     mapping (address => Module) internal allModules;
 
+    // List of custom functions indexed by signature
+    mapping (bytes4 => address) internal customFunctions;
```

Comment:
Could we make this public to aid debugging?

Reply:
Addressed in 3e5d18a6eeb5c9e20f57eacea79ba35151820a98